issue-labels.md

```diff
@@ -0,0 +1,50 @@
+# Issue Labels
+
+GitHub Issues are tagged with a standard set of labels across all kubeflow repositories, as defined here. These are used as part of the automated PR merge process, as well as tracking project milestones and releases.
+
+## List of labels
+
+| Category    | Label
+| :--------   | :----
+| *none*      | do-not-merge
+|             | duplicate
+|             | high-impact
+|             | starter
+|             | testing
+|             | wont-fix
+| addition    | feature
+| area        | api
+|             | build-release
+|             | docs
+|             | example
+|             | front-end
+|             | logging
+|             | lifecycle
+|             | operator
+| community   | discussion
+|             | question
+| improvement | enhancement
+|             | optimization
+| os          | centos
+|             | macos
+|             | ubuntu
+|             | windows
+| platform    | aws
+|             | azure
+|             | gcp
+|             | minikube
+|             | other
+| priorities  | p0
+|             | p1
+|             | p2
+| problems    | bug
+|             | refactor
+|             | security
+| status      | icebox
+|             | backlog
+|             | in progress
```

Comment:
Why use whitespace in `in progress` when hypens are used for other labels

Reply:
Good catch